compiler/q-implant/README.md

```diff
@@ -0,0 +1,25 @@
+# q-implant
+
+_q-implant_ writes quantization parameters and weights (given as .json and .npy files) to a circle model.
+
+## Example
+
+```sh
+q-implant input.circle qparam.json output.circle
+```
+
+`qparam.json` and `*.npy` files must exist in the same directory. For example,
+
+```json
+{
+  "ifm": {
+    "dtype": "uint8",
+    "scale": "0.npy",
+    "zerop": "1.npy",
```

Comment:
above sentence used word `weights` but these don't seem to be weights.

Reply:
can you add a sample json with input, conv2d(with kernel and bias), output ?

Reply:
> above sentence used word `weights` but these don't seem to be weights.

This README.md was created assuming that "q-implant" was applied to a simple Conv2D model. This model consists of `ifm`, `weight`, and `bias` tensor as input, and `ofm` tensor as output.

In the case of `ifm` written as an example on README.md, it is a tensor that does not include value, so there is no part to write weights, but only quantization parameters.

There seems to be a lack of explanation for this part. We'll add an example for a tensor with value.

Reply:
>can you add a sample json with input, conv2d(with kernel and bias), output ?

this is sample used in q-implant-op-level-test 

1. input file : Generate circle files from TFLite recipes
"TFLite recipe" -> tflchef -> "tflite file" -> tflite2circle -> "circle file"

2. json file
```
{
  "ifm": {
    "dtype": "uint8",
    "scale": "0.npy",
    "zerop": "1.npy",
    "quantized_dimension": 0
  },
  "ker": {
    "dtype": "uint8",
    "scale": "2.npy",
    "zerop": "3.npy",
    "quantized_dimension": 0,
    "value": "4.npy"
  },
  "bias": {
    "dtype": "int32",
    "scale": "5.npy",
    "zerop": "6.npy",
    "quantized_dimension": 0,
    "value": "7.npy"
  },
  "ofm": {
    "dtype": "uint8",
    "scale": "8.npy",
    "zerop": "9.npy",
    "quantized_dimension": 0
  }
}
```

3. output file : dump to use circledump
```
Dump: /home/ssafy/ONE/ONE/build/compiler/q-implant-op-level-test/Conv2D_000_Q8/output.circle

===================================================================
Model version: 0
 # sub graphs: 1

Operator Codes: [order] OpCodeName (OpCode Enum)
[0] CONV_2D (code: 3, dep_code: 3, version: 1)

Buffers: B(index) (length) values, if any
B(0) (0) 
B(1) (0) 
B(2) (2) 0x6e 0x80
B(3) (4) 0x36 00 00 00
B(4) (0) 

metadata : B(index) name

-------------------------------------------------------------------
Sub-Graph: #0 main

Data Format:
CHANNEL_LAST (NHWC for 2d, NDHWC for 3d data)

Operands: T(subgraph index : tensor index) TYPE (shape) (shape_signature) B(buffer index) (variable) OperandName
T(0:0) UINT8 (1, 3, 3, 2) B(1) ifm
    Quantization: scale(0.0686598) zeropt(169) quantized_dimension(0)

T(0:1) UINT8 (1, 1, 1, 2) B(2) ker
    Quantization: scale(0.582998) zeropt(58) quantized_dimension(0)

T(0:2) INT32 (1) B(3) bias
    Quantization: scale(0.543995) zeropt(164) quantized_dimension(0)

T(0:3) UINT8 (1, 3, 3, 1) B(4) ofm
    Quantization: scale(0.549946) zeropt(35) quantized_dimension(0)

Operators: O(subgraph index : operator index) OpCodeName 
    Option(values) ... <-- depending on OpCode
    I T(tensor index) OperandName <-- as input
    O T(tensor index) OperandName <-- as output
O(0:0) CONV_2D 
    Padding(1) Stride.W(1) Stride.H(1) Dilation.W(1) Dilation.H(1) Activation(NONE)
    I T(0:0) ifm
    I T(0:1) ker
    I T(0:2) bias
    O T(0:3) ofm

Inputs/Outputs: I(input)/O(output) T(tensor index) OperandName
I T(0:0) ifm
O T(0:3) ofm

===================================================================
```

Reply:
> this is sample used in q-implant-op-level-test

not here ... in to the README.md file.